taxonomies/allergens.txt

```diff
@@ -428,11 +428,11 @@ nl:weekdieren
 nl_be:weekdieren
 pl:mięczaki
 pt:moluscos
-ro:moluște
 ro:моллюск, моллюски, кальмар, каракатица, устрицы, устрица, ракушки, гребешок, улитка, улитки
```

Comment:
@lucaa can you confirm that cc @stephanegigandet

Reply:
It's greek to me.

Reply:
Or Russian :)